src/Illuminate/Console/GeneratorCommand.php

```diff
@@ -146,7 +146,7 @@ public function handle()
         if ((! $this->hasOption('force') ||
              ! $this->option('force')) &&
              $this->alreadyExists($this->getNameInput())) {
-            $this->error($this->type.' already exists!');
+            $this->error("{$this->type} {$path} already exists!");
```

Comment:
For my use case, I don't have a need for showing the path when it already exists, although some tools, such as `
vim-laravel` do offer a "Open if exists" option, so it probably would be useful here as well.

I would suggest leaving the existing message, but on the following line add...

```php
$this->line('Path: '.$path);
```